__tests__/routes/share.test.js

```diff
@@ -0,0 +1,147 @@
+import request from 'supertest';
+import { Op } from 'sequelize';
+import bcrypt from 'bcrypt';
+import app from '../../app';
+import { urlPrefix } from '../mocks/variables.json';
+import { User, Article } from '../../database/models';
+import { createArticle2, signupUser } from '../mocks/db.json';
+
+let loginUser1;
+let loginUser2;
```

Comment:
@mcaleb808 loginUser2 is not necessary as it is not being used.

Reply:
I am using it!